src/electionguard/encrypt.py

```diff
@@ -321,7 +327,11 @@ def encrypt_contest(
         # false is entered instead and the selection_count is not incremented
         # this allows consumers to only pass in the relevant selections made by a voter
         for selection in contest.ballot_selections:
-            if selection.object_id == description.object_id:
+            # If overvote, no votes should be counted and instead placeholders should be used.
+            if (
+                selection.object_id == description.object_id
+                and error is not ContestErrorType.OverVote
```

Comment:
This is a quick sidestep to avoid dramatically changing the loops in this section. 

Essentially, if an over vote has been detected, none of the selections from the ballot are appended. This means empty selections as per the descriptions from the manifest will be used instead.